src/applications/model/gaming-streaming-server.h

```diff
@@ -92,34 +88,41 @@ class GamingStreamingServer : public Application
   void EraseStatistics (void);
 
 protected:
+  /**
+   * \brief Add parameters of new stream
+   *
+   * \param packetSize A random number generator for packet size
+   * \param interArrivalTime A random number generator for inter-arrival time
+   */
+  void AddNewTraffic (Ptr<RandomVariableStream> packetSize, Ptr<RandomVariableStream> interArrivalTime);
+
   /**
    * \brief Generate and send packets based on the different distribution
    *
-   * \param pktDist random variable which defines the size of generated packets
-   * \param iatDist random variable which defines inter-arrival times between the packets
+   * \param packetSize random variable which defines the size of generated packets
+   * \param interArrivalTime random variable which defines inter-arrival times between the packets
    */
-  void SendPacket (Ptr<RandomVariableStream> pktDist, Ptr<RandomVariableStream> iatDist);
+  void Send (Ptr<RandomVariableStream> packetSize, Ptr<RandomVariableStream> interArrivalTime);
+
+  /**
+   *  Initialize the parameters of different streams
+   */
+  virtual void InitializeStreams (void);
+
+private:
 
-  uint32_t     m_sentSeq;  //!< Sequence number for packets
+  uint32_t     m_seq;  //!< Sequence number for packets
   uint32_t     m_totSentPackets; //!< Counter for sent packets
   Ptr<Socket>  m_socket; //!< Socket
-  EventId      m_sendEvent; //!< Event to send the next packet
```

Comment:
why was this removed? how do you stop the app?

Reply:
Actually having a single EventId doesn't help. 
I need to save a vector of EventId's (one for each stream).